### PR TITLE
Add coreos/bootupd

### PR DESCRIPTION
**Pull request by cgwalters**
Mostly following what's there for rpm-ostree today.

### CODE REVIEW

File: ci-operator/config/coreos/bootupd/OWNERS [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+- ashcrow
+- cgwalters
+- jlebon
+- miabbott
+options: {}
+reviewers:
+- ashcrow
+- cgwalters
+- jlebon
+- miabbott
```
Comment (miabbott) [inline ci-operator/config/coreos/bootupd/OWNERS:14]: What about expanding these lists to include @darkmuggle and @lucab?

Comment (cgwalters) [reply ci-operator/config/coreos/bootupd/OWNERS:14]: Sure, though I think we should do that as a second step; this is just copying existing files.

Comment (cgwalters) [reply ci-operator/config/coreos/bootupd/OWNERS]: OK looking around I found out we can use symlinks for `OWNERS`, so I did that for now.

**Review by petr-muller — APPROVED** [review]

**Comment by cgwalters** [comment]
OK so honestly I get a bit lost in all the copy/paste YAML going on here and I seem to have messed something up because Prow is handling `/approve` for
https://github.com/coreos/bootupd/pull/17
but not handling `/lgtm` or running the presubmit.  Anyone have an idea?

**Pull request merged**